elephant/spike_train_generation.py

```diff
@@ -17,6 +17,94 @@
 import random
 from elephant.spike_train_surrogates import dither_spike_train
 
+def spike_extraction(signal, threshold = 0.0 * mV, sign = 'above',
```

Comment:
From PEP8: "Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value."